app/models/item.rb

```diff
@@ -25,4 +28,15 @@ def no_orders?
     item_orders.empty?
   end
 
+  def total_sold
+    item_orders.sum(:quantity)
+  end
+
+  def self.most_popular_list
+    join_with_item_orders.group_by_quantity.order("order_quantity DESC").limit(5)
```

Comment:
This is looking good. Cool use of the `scope` helper keyword

Reply:
Agreed! Thanks for introducing me to this.